v4/export/config.go

```diff
@@ -49,46 +49,48 @@ type Config struct {
 	CsvSeparator  string
 	CsvDelimiter  string
 
-	TableFilter       filter.Filter
-	Rows              uint64
-	Where             string
-	FileType          string
-	EscapeBackslash   bool
-	DumpEmptyDatabase bool
-	SessionParams     map[string]interface{}
+	TableFilter          filter.Filter
+	Rows                 uint64
+	Where                string
+	FileType             string
+	EscapeBackslash      bool
+	DumpEmptyDatabase    bool
+	OutputFilenameFormat string
```

Comment:
consider using the parsed `*template.Template` here so it does not need to be parsed every time we call `WriteTableData`.

Reply:
addressed in https://github.com/pingcap/dumpling/pull/120/commits/50d18ad53396b887b1cbf08309eaeb153dd48638